server/amqp.go

```diff
@@ -55,6 +57,10 @@ type amqpTransport struct {
 	address       string
 }
 
+func init() {
+	transports["aqmp"] = amqpTransportFromConfig
```

Comment:
aqmp -> amqp

Reply:
Oh. *Again*? I'm transposing those letters consistently. It must be something wrong with my brain.

I'll grep through the codebase just to make sure.